modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.cpp.mustache

```diff
@@ -427,8 +444,65 @@ QByteArray {{prefix}}HttpRequestWorker::decompress(const QByteArray& data){
             sts = true;
         } while (strm.avail_out == 0);
         inflateEnd(&strm);
-    } while(false);{{/contentCompression}}
-    return sts ? result : QByteArray();
+    } while(false);
+    return sts ? result : QByteArray();{{/contentCompression}}{{^contentCompression}}
+    Q_UNUSED(data);
+    return QByteArray();{{/contentCompression}}
+}
+
+QByteArray {{prefix}}HttpRequestWorker::compress(const QByteArray& input, int level, {{prefix}}CompressionType compressType) {
+    {{#contentCompression}}QByteArray output;
+    static const int GZIP_WINDOWS_BIT = 15+16;
```

Comment:
Are these called 'windows bit' or 'window bit'?

Reply:
It is *window* bit